.github/workflows/build_windows_packages.yml

```diff
@@ -100,10 +118,14 @@ jobs:
           path: amdgpu-windows-interop
           lfs: true
 
-      - name: Configure MSVC
-        uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
+      - name: Install python deps
+        run: |
+          pip install -r requirements.txt
 
       - name: Configure Projects
+        env:
+          CC: ${{ env.VCToolsInstallDir }}/bin/Hostx64/x64/cl.exe   # Set to MSVC explicitly
+          CXX: ${{ env.VCToolsInstallDir }}/bin/Hostx64/x64/cl.exe  # Set to MSVC explicitly
```

Comment:
Note to self: check if this (or the perl install) affects ccache. Might get cache misses on the first few builds, then catch back up? Could increment the cache key version.